### ISSUE TITLE
Salesforce CLI fails to install LWC Local Development Server

### ISSUE THREAD

**Issue by MainakKarmakar17** [forcedotcom/cli#2980]
> **Note** 
> Before you submit your issue, make sure that:
> - You're using the latest version of Salesforce CLI.
> - You've searched both open and closed issues for related posts.
> - You've used the `doctor` command to diagnose common issues. 
> - You understand that GitHub Issues don't adhere to any agreement or SLA.
>   - If you require immediate assistance, use official channels such as Salesforce Customer Support.

### Summary
I am trying to install the local development server on my machine but it fails with errno: "-4071" and code: "EINVAL"

### Steps To Reproduce

1. I have installed Salesforce CLI with npm, to try out this local development server installation. (Previously I had CLI standard cli installer which I used to carry out my development work). I deleted my previous setup of cli, vs code and node js and decided to install with NPM to see if installation of local dev server would work.
2. I have latest version of cli @salesforce/cli/2.52.6 win32-x64 node-v18.20.4
3. I have VS Code latest version
4. I have installed npm version 18.20.4 and npm 10.7.0
5. I am running this command sf plugins install @salesforce/lwc-dev-server

### Expected result
The local development server should be installed.

### Actual result
PS C:\Salesforce Projects\AdvancedApexSpecialist Project> sf plugins install @salesforce/lwc-dev-server  
Polling for new version(s) to become available on npm... done
Successfully validated digital signature for @salesforce/lwc-dev-server.
Finished digital signature check.
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated @babel/plugin-proposal-class-properties@7.14.5: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
npm warn deprecated @babel/plugin-proposal-class-properties@7.1.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
npm warn deprecated @babel/plugin-proposal-object-rest-spread@7.0.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
npm warn deprecated @babel/plugin-proposal-object-rest-spread@7.14.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
npm warn deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm warn deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm warn deprecated rimraf@2.4.5: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@2.6.3: Rimraf versions prior to v4 are no longer supported
npm warn deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm warn deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm warn deprecated har-validator@5.1.5: this library is no longer supported
npm warn deprecated @oclif/screen@1.0.4: Deprecated in favor of @oclif/core
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@6.0.4: Glob versions prior to v9 are no longer supported
npm warn deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm warn deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm warn deprecated q@1.5.1: You or someone you depend on is using Q, the JavaScript Promise library that gave JavaScript developers strong feelings about promises. They can almost certainly migrate to the native JavaScript promise now. Thank you literally everyone for joining me in this bet against the odds. Be excellent to each other.
npm warn deprecated
npm warn deprecated (For a CapTP with native promises, see @endo/eventual-send and @endo/captp)
npm warn deprecated rollup-plugin-replace@2.2.0: This module has moved and is now available at @rollup/plugin-replace. Please update your dependencies. This version is no longer maintained.
npm warn deprecated domexception@1.0.1: Use your platform's native DOMException instead
npm warn deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm warn deprecated rollup-plugin-node-resolve@4.2.4: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-node-resolve.
npm warn deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm warn deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm warn deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm warn deprecated @oclif/command@1.8.36: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm warn deprecated @oclif/help@1.0.15: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm warn deprecated uuid@3.3.3: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm warn deprecated rollup-plugin-commonjs@9.3.4: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-commonjs.
npm warn deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm warn deprecated @salesforce/command@3.1.3: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @oclif/errors@1.3.6: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm warn deprecated fancy-test@1.4.10: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @oclif/config@1.18.17: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @oclif/config@1.18.16: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @oclif/parser@3.8.17: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated cli-ux@4.9.3: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @lwc/engine@1.2.2: Use @lwc/engine-dom instead
npm warn deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
npm warn deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
npm warn cleanup Failed to remove some directories [
npm warn cleanup   [
npm warn cleanup     'C:\\Users\\MAINAK KARMAKAR\\AppData\\Local\\sf\\node_modules',
npm warn cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\@salesforce\lwc-dev-server-dependencies\vendors\dependencies-226\lightning-pkg\src'] {
npm warn cleanup       errno: -4048,
npm warn cleanup       code: 'EPERM',
npm warn cleanup       syscall: 'rmdir',
npm warn cleanup       path: 'C:\\Users\\MAINAK KARMAKAR\\AppData\\Local\\sf\\node_modules\\@salesforce\\lwc-dev-server-dependencies\\vendors\\dependencies-226\\lightning-pkg\\src'
npm warn cleanup     }
npm warn cleanup   ]
npm warn cleanup ]
npm error code 1
npm error path C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers
npm error command failed
npm error command C:\Windows\system32\cmd.exe /d /s /c node build.js || nodejs build.js
npm error node:internal/child_process:421
npm error     throw new ErrnoException(err, 'spawn');
npm error     ^
npm error
npm error Error: spawn EINVAL
npm error     at ChildProcess.spawn (node:internal/child_process:421:11)
npm error     at Object.spawn (node:child_process:761:9)
npm error     at build (C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers\build.js:58:5)
npm error     at Object.<anonymous> (C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers\build.js:47:3)
npm error     at Module._compile (node:internal/modules/cjs/loader:1358:14)
npm error     at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
npm error     at Module.load (node:internal/modules/cjs/loader:1208:32)
npm error     at Module._load (node:internal/modules/cjs/loader:1024:12)
npm error     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
npm error     at node:internal/main/run_main_module:28:49 {
npm error   errno: -4071,
npm error   code: 'EINVAL',
npm error   syscall: 'spawn'
npm error }
npm error
npm error Node.js v20.16.0
npm error 'nodejs' is not recognized as an internal or external command,
npm error operable program or batch file.
npm error A complete log of this run can be found in: C:\Users\MAINAK KARMAKAR\AppData\Local\npm-cache\_logs\2024-08-03T08_28_58_979Z-debug-0.log
@salesforce/cli: Installing plugin @salesforce/lwc-dev-server@latest... failed
 »   Error: node "C:\Users\MAINAK KARMAKAR\AppData\Roaming\npm\node_modules\@salesforce\cli\node_modules\npm\bin\npm-cli.js" install
 »   @salesforce/lwc-dev-server@latest --omit dev --no-audit --loglevel=notice --no-fund exited with code 1
 »   Try this: Run with DEBUG=@oclif/plugin-plugins* to see debug output.
PS C:\Salesforce Projects\AdvancedApexSpecialist Project> node -v                                      
v18.20.4
PS C:\Salesforce Projects\AdvancedApexSpecialist Project> npm -v
10.7.0
PS C:\Salesforce Projects\AdvancedApexSpecialist Project> sf plugins install @salesforce/lwc-dev-server
Polling for new version(s) to become available on npm... done
Successfully validated digital signature for @salesforce/lwc-dev-server.
Finished digital signature check.
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated @babel/plugin-proposal-class-properties@7.14.5: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
npm warn deprecated @babel/plugin-proposal-class-properties@7.1.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.
npm warn deprecated @babel/plugin-proposal-object-rest-spread@7.0.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
npm warn deprecated @babel/plugin-proposal-object-rest-spread@7.14.7: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-object-rest-spread instead.
npm warn deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm warn deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm warn deprecated rimraf@2.4.5: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@2.6.3: Rimraf versions prior to v4 are no longer supported
npm warn deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm warn deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm warn deprecated har-validator@5.1.5: this library is no longer supported
npm warn deprecated @oclif/screen@1.0.4: Deprecated in favor of @oclif/core
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@6.0.4: Glob versions prior to v9 are no longer supported
npm warn deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm warn deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm warn deprecated q@1.5.1: You or someone you depend on is using Q, the JavaScript Promise library that gave JavaScript developers strong feelings about promises. They can almost certainly migrate to the native JavaScript promise now. Thank you literally everyone for joining me in this bet against the odds. Be excellent to each other.
npm warn deprecated
npm warn deprecated (For a CapTP with native promises, see @endo/eventual-send and @endo/captp)
npm warn deprecated rollup-plugin-replace@2.2.0: This module has moved and is now available at @rollup/plugin-replace. Please update your dependencies. This version is no longer maintained.
npm warn deprecated domexception@1.0.1: Use your platform's native DOMException instead
npm warn deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm warn deprecated rollup-plugin-node-resolve@4.2.4: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-node-resolve.
npm warn deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm warn deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm warn deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm warn deprecated @oclif/command@1.8.36: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm warn deprecated @oclif/help@1.0.15: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm warn deprecated uuid@3.3.3: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm warn deprecated rollup-plugin-commonjs@9.3.4: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-commonjs.
npm warn deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm warn deprecated @salesforce/command@3.1.3: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @oclif/errors@1.3.6: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm warn deprecated fancy-test@1.4.10: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @oclif/config@1.18.17: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @oclif/config@1.18.16: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @oclif/parser@3.8.17: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated cli-ux@4.9.3: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm warn deprecated @lwc/engine@1.2.2: Use @lwc/engine-dom instead
npm warn deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
npm warn deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
npm warn cleanup Failed to remove some directories [
npm warn cleanup   [
npm warn cleanup     'C:\\Users\\MAINAK KARMAKAR\\AppData\\Local\\sf\\node_modules\\@salesforce\\lwc-dev-server-dependencies',
npm warn cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\@salesforce\lwc-dev-server-dependencies\vendors\dependencies-228\lightning-stub-pkg\modules\lightning-stubs'] {
npm warn cleanup       errno: -4048,
npm warn cleanup       code: 'EPERM',
npm warn cleanup       syscall: 'rmdir',
npm warn cleanup       path: 'C:\\Users\\MAINAK KARMAKAR\\AppData\\Local\\sf\\node_modules\\@salesforce\\lwc-dev-server-dependencies\\vendors\\dependencies-228\\lightning-stub-pkg\\modules\\lightning-stubs'
npm warn cleanup     }
npm warn cleanup   ]
npm warn cleanup ]
npm error code 1
npm error path C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers
npm error command failed
npm error command C:\Windows\system32\cmd.exe /d /s /c node build.js || nodejs build.js
npm error node:internal/child_process:414
npm error     throw errnoException(err, 'spawn');
npm error     ^
npm error
npm error code 1
npm error path C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers
npm error command failed
npm error command C:\Windows\system32\cmd.exe /d /s /c node build.js || nodejs build.js
npm error node:internal/child_process:414
npm error     throw errnoException(err, 'spawn');
npm error     ^
npm error
npm error     throw errnoException(err, 'spawn');
npm error     ^
npm error
npm error Error: spawn EINVAL
npm error     at ChildProcess.spawn (node:internal/child_process:414:11)
npm error     at Object.spawn (node:child_process:761:9)
npm error     at build (C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers\build.js:58:5)
npm error     at Object.<anonymous> (C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers\build.js:47:3)
npm error Error: spawn EINVAL
npm error     at ChildProcess.spawn (node:internal/child_process:414:11)
npm error     at Object.spawn (node:child_process:761:9)
npm error     at build (C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers\build.js:58:5)
npm error     at Object.<anonymous> (C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers\build.js:47:3)
npm error     at Module._compile (node:internal/modules/cjs/loader:1364:14)
npm error     at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
npm error     at Module.load (node:internal/modules/cjs/loader:1203:32)
npm error     at Module._load (node:internal/modules/cjs/loader:1019:12)
npm error     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
npm error     at node:internal/main/run_main_module:28:49 {
npm error     at build (C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers\build.js:58:5)
npm error     at Object.<anonymous> (C:\Users\MAINAK KARMAKAR\AppData\Local\sf\node_modules\fibers\build.js:47:3)
npm error     at Module._compile (node:internal/modules/cjs/loader:1364:14)
npm error     at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
npm error     at Module.load (node:internal/modules/cjs/loader:1203:32)
npm error     at Module._load (node:internal/modules/cjs/loader:1019:12)
npm error     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
npm error     at node:internal/main/run_main_module:28:49 {
npm error   errno: -4071,
npm error     at Module._compile (node:internal/modules/cjs/loader:1364:14)
npm error     at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
npm error     at Module.load (node:internal/modules/cjs/loader:1203:32)
npm error     at Module._load (node:internal/modules/cjs/loader:1019:12)
npm error     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
npm error     at node:internal/main/run_main_module:28:49 {
npm error   errno: -4071,
npm error   errno: -4071,
npm error   code: 'EINVAL',
npm error   code: 'EINVAL',
npm error   syscall: 'spawn'
npm error }
npm error
npm error Node.js v18.20.4
npm error 'nodejs' is not recognized as an internal or external command,
npm error operable program or batch file.
npm error A complete log of this run can be found in: C:\Users\MAINAK KARMAKAR\AppData\Local\npm-cache\_logs\2024-08-03T10_37_31_193Z-debug-0.log
@salesforce/cli: Installing plugin @salesforce/lwc-dev-server@latest... failed
 »   Error: node "C:\Users\MAINAK KARMAKAR\AppData\Roaming\npm\node_modules\@salesforce\cli\node_modules\npm\bin\npm-cli.js" install
 »   @salesforce/lwc-dev-server@latest --omit dev --no-audit --loglevel=notice --no-fund exited with code 1
 »   Try this: Run with DEBUG=@oclif/plugin-plugins* to see debug output.

### System Information
I have latest version of cli @salesforce/cli/2.52.6 win32-x64 node-v18.20.4
I have VS Code latest version
I have installed node version 18.20.4 and npm 10.7.0
I am running this command sf plugins install @salesforce/lwc-dev-server

```json
### Additional information
<!-- Feel free to attach a screenshot -->

**Comment by MainakKarmakar17** [comment]
SF CLI version
{
  "architecture": "win32-x64",
  "cliVersion": "@salesforce/cli/2.52.6",
  "nodeVersion": "node-v18.20.4",
  "osVersion": "Windows_NT 10.0.22631",
  "rootPath": "C:\\Users\\MAINAK KARMAKAR\\AppData\\Roaming\\npm\\node_modules\\@salesforce\\cli",
  "shell": "powershell",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 3.1.9 (core)",
    "@oclif/plugin-commands 4.0.9 (core)",
    "@oclif/plugin-help 6.2.7 (core)",
    "@oclif/plugin-not-found 3.2.13 (core)",
    "@oclif/plugin-plugins 5.3.9 (core)",
    "@oclif/plugin-search 1.2.4 (core)",
    "@oclif/plugin-update 4.4.12 (core)",
    "@oclif/plugin-version 2.2.9 (core)",
    "@oclif/plugin-warn-if-update-available 3.1.10 (core)",
    "@oclif/plugin-which 3.2.9 (core)",
    "@salesforce/cli 2.52.6 (core)",
    "apex 3.3.2 (core)",
    "auth 3.6.38 (core)",
    "data 3.5.8 (core)",
    "deploy-retrieve 3.9.21 (core)",
    "info 3.3.21 (core)",
    "limits 3.3.20 (core)",
    "marketplace 1.2.19 (core)",
    "org 4.4.0 (core)",
    "packaging 2.7.4 (core)",
    "schema 3.3.21 (core)",
    "settings 2.3.9 (core)",
    "sobject 1.4.20 (core)",
    "source 3.5.7 (core)",
    "telemetry 3.6.3 (core)",
    "templates 56.3.6 (core)",
    "trust 3.7.15 (core)",
    "user 3.5.20 (core)"
  ]
}

**Comment by mdonnalley** [comment]
There are a number of open bugs about not being able to install lwc-dev-server: https://github.com/forcedotcom/lwc-dev-server-feedback/issues

Unfortunately, it's deprecated and its replacement is still under development
